packages/tailwindcss/src/index.ts

```diff
@@ -188,9 +191,307 @@ async function parseCss(
         )
       }
 
-      customUtilities.push((designSystem) => {
-        designSystem.utilities.static(name, () => structuredClone(node.nodes))
-      })
+      // Functional utilities. E.g.: `tab-size-*`
```

Comment:
I wonder if it's a good time to pull this stuff out into a separate file? This is a fairly large chunk of code.